config/initializers/dfe_signin.rb

```diff
@@ -0,0 +1,7 @@
+DFE_SIGN_IN_ROLE = ENV.fetch("DFE_SIGN_IN_ROLE")
```

Comment:
As mentioned above, let's define this constant closer to where it is used, rather than as a global constant.